docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+    - SQL (e.g. PostgreSQL, MySQL)
+    - SQL - embedded (e.g. SQLite)
+    - NoSQL - columnar store (e.g. Parquet, Arrow, HDFS, BigQuery).
+    - NoSQL - document store (e.g. MongoDB, Elasticsearch, CouchDB).
+    - Graph database (Neo4j, TigerGraph).
+    - Time Series (e.g. InfluxDB).
+    - Pub sub (e.g. Kafka, Druid).
+    - Key value (e.g. Redis, MemcacheDB).
+    - Google Sheets.
+    - ❗Industry or field specific APIs.
+    - Streaming.
+    - Other (please specify)
+
+9. What data formats are you mostly working with? (pick up to 3)
+    - Tabular (csv, spreadsheet, database tables, parquet).
+    - Images.
+    - Tensors (deep learning).
+    - Nested (JSON, NoSQL).
+    - Hierarchical Data Format (HDF5).
+    - Time series.
+    - Text.
+    - Audio.
+    - Video.
+    - 3D/ CAD.
+    - Graph (nodes, edges).
+    - Spatial/ geographic (coordinates, GIS).
+    - Game/ reinforcement simulation.
+    - ❗Industry-specific file formats.
+    - Other (please specify)
+
+10. Do you experience these problems with data in Jupyter? (rate from scale of 0-4)
+    - Data is too big to fit into memory on my machine/ server.
+    - Can’t see a list of my current variables.
+    - Poor MVC/ ORM integrations (Django, Flask).
+    - Managing database/ source connections and secrets.
+    - No grid view for manipulating/ filtering dataframes and arrays.
+    - Lost data during failure or restart of kernel/ server.
+    - Other (please specify)
+    
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+11. What type of analysis are you running? (up to 4)
+    - ❗I am not performing ML/statistical tasks.
+    - Regression (predict a numeric output).
+    - Classification (predict a categorical output).
+    - Generative/ auto-encode (create new data based on existing data).
+    - Reinforcement learning (actions that maximize a reward)
+    - Dimensionality reduction (PCA, K-Nearest Neighbors)
+    - Feature engineering (importance, extraction, selection, permutation).
+    - Natural language processing.
+    - Graph data science.
+    - Outlier detection.
+    - Other (please specify)
+    
+    
+### Visualization
+
+12. What tools does your team use to create dashboards tools? (pick up to 3)
+    - ❗My team doesn’t create dashboard tools.
+    - Write our own in HTML & JS.
+    - R Shiny.
+    - Kibana.
+    - Dash-Plotly.
+    - Voila.
+    - Tableau.
+    - Looker.
+    - Klipfolio.
+    - Google Data Studio.
+    - Spotfire.
+    - Grafana
+    - Other (please specify)
+
+13. Do you experience these problems with visualization in Jupyter?
+    - No built-in UI for creating charts.
+    - Can't publish my charts as web-based dashboards.
+    - Poor/ buggy support for my plotting tool.
+    - Displaying highly dimensional data (array of array of arrays, too many rows/ columns to fit on screen).
```

Comment:
All the other questions seem to be written from a "negative"  point of view, except this one

Reply:
☑️ changes integrated to each problem-style question